packages/ckeditor5-engine/src/controller/datacontroller.js

```diff
@@ -90,7 +90,8 @@ export default class DataController {
 		 * @member {module:engine/conversion/downcastdispatcher~DowncastDispatcher}
 		 */
 		this.downcastDispatcher = new DowncastDispatcher( {
-			mapper: this.mapper
+			mapper: this.mapper,
```

Comment:
The new param needs to be documented.

Reply:
I am not sure if we document `conversionApi` "content" anywhere.